db/schema.rb

```diff
@@ -342,7 +343,7 @@
     t.string "preferred_language", default: "tr"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.datetime "password_changed_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "password_changed_at", default: -> { "now()" }, null: false
```

Comment:
Bunu test ettik değil mi?

Reply:
OK aynıymış: https://dba.stackexchange.com/questions/63548/difference-between-now-and-current-timestamp